site/docs/components/button/examples.mdx

```diff
@@ -9,57 +9,52 @@ data:
 ---
 
 <LivePreviewControls>
-<LivePreview componentName="button" exampleName="CTA" >
+  
+<LivePreview componentName="button" exampleName="Appearance">
 
-## CTA
+## Appearance
 
-Use the CTA button for high-priority actions. For example, a CTA button may prompt a user to register, sign up, submit or buy.
-
-### Best practices
-
-Use CTA buttons sparingly. They represent the default or expected action on the page.
+Buttons come in three distinct appearances: solid, bordered, and transparent. These styles provide greater choice and flexibility for establishing hierarchy and affordance within a UI.
 
 </LivePreview>
 
-<LivePreview componentName="button" exampleName="Primary" >
-
-## Primary
-
-This is the default variant. Use the primary button for routine, nonurgent actions, for example, to move to the next page or complete a task.
+<LivePreview componentName="button" exampleName="Sentiment">
 
-</LivePreview>
-
-<LivePreview componentName="button" exampleName="Secondary" >
+## Sentiment
 
-## Secondary
+The buttons convey five sentiments through color—neutral, accented, negative, caution, and positive. Use them in your application designs as follows:
 
-Use the secondary button for noncritical actions that support the user but don't impact a flow. They're typically the alternative option available alongside a primary action. For example, a secondary “back” button may support a primary “next” button.
+- **Neutral**: Ideal for routine, non-urgent actions, or where the actions should blend in with the UI.
```

Comment:
@mark-tate, you mentioned a potential specific order for the sentiments. Are you happy with this order?

Reply:
it should be seen as a ramp, as otherwise they are open to interpretation and you just pick the color you like the look of. 

So a ramp would be

Negative
Caution
Neutral
Accented
Positive

Reply:
I think listing them in a logical order (what you suggest) makes sense but i agree with Zhihao it's a logical order, not necessarily a ramp with directly opposing values. If we document in that order with clear definitions, that should be sufficient.